content/no-code.md

```diff
@@ -106,73 +102,73 @@ The persona scenarios below are examples of accessibility issues with no-code to
       </q></span></p>
     </li>
   </ul>
 </div>
 
-Your users may not be able to fix accessibility issues themselves. Your tool needs to support them when creating content so that the output is accessible. This includes making headings, paragraphs, lists, labels, and other content structures accessible.
+The templates, components, and pre-authored content that your tool provides needs to be accessible. That includes making headings, lists, labels, and other content structures accessible. When your tool offers customization options, such as colors, help content creators make accessible choices.
 
-### The tool supports the creation of accessible multimedia content
+### The tool helps make images and videos accessible
 
 <p><span class="persona">Persona: </span>
-  Dacso runs a mechanic shop. He has a website where he puts videos that give customers short tips for fixing cars and motorbikes.
+  Dacso runs an automotive repair shop and he does a lot of body work. His daughter created a website for him to showcase his work. Dacso uploads before-and-after photos of his projects. He also posts videos with simple maintenance tips.
 </p>
 <div class="quotes">
   <ul>
     <li>
       <p><span class="issue">Problem: </span><span><q>
-        I have been told that my videos need captions, but I don't see how to add them with my website tool.
+        I usually get lots of great feedback on my website. A couple of Deaf people say they want to learn from my videos, but of course they can't hear what I'm saying. I really want to make it so they can get advice from my videos, but I have no idea what to do.
       </q></span></p>
     </li>
     <li>
       <p><span class="issue">Works well: </span><span><q>
-        Whenever I upload a video, my tool alerts me to add captions. They also have articles and tips that suggested I also add transcripts and how to describe the video.
+        Whenever I upload a picture, the tool asks me for a text description. It has "more info" that explained how to write a good text description for people who are blind and can't see the image. It also has good support for adding captions to videos. And the "more info" for videos explained how to describe what I'm doing when I make the video, for people who don't see too well.
       </q></span></p>
     </li>
   </ul>
 </div>
 
-Your users may not have the knowledge and/or resources to create accessible multimedia content for people who cannot see or hear well or at all. Your tool needs to support the addition of alternatives for audio and video content so that people with disabilities can enjoy them.
+Your tool should enable and encourage content creators to provide alternative text for images and for videos. It should include guidance on how to provide useful alternatives for their website consumers who have disabilities.
```

Comment:
```suggestion
Your tool should enable and encourage content creators to provide alternative text for images, audio, and videos. It should include guidance on how to provide useful alternatives for their website consumers who have disabilities.
```

I think otherwise it could be read as if audio-only does not need alternative.